LemonadeGui.py

```diff
@@ -182,6 +210,15 @@ def draw(self, screen, tick):
                  block.get_width() /
                  2,
                  0))
+        if self.help:
+            block = self.draw_help()
+            screen.blit(
+                block,
+                (self.game_engine.width /
+                 2 -
+                 block.get_width() /
+                 2,
+                 0))
```

Comment:
This is unreadable and not pep8 compliant.

Reply:
I used the same format as it was used in the previous screen.blit() to maintain constistency, I agree not good idea, fixed.